react/features/base/participants/middleware.js

```diff
@@ -87,17 +87,13 @@ MiddlewareRegistry.register(store => next => action => {
         store.dispatch(localParticipantIdChanged(action.conference.myUserId()));
         break;
 
-    case DOMINANT_SPEAKER_CHANGED: {
-        // Lower hand through xmpp when local participant becomes dominant speaker.
-        const { id } = action.participant;
+    case LOCAL_PARTICIPANT_AUDIO_LEVEL_CHANGED: {
         const state = store.getState();
         const participant = getLocalParticipant(state);
-        const isLocal = participant && participant.id === id;
 
-        if (isLocal && hasRaisedHand(participant) && !getDisableRemoveRaisedHandOnFocus(state)) {
+        if (action.level > 0.009 && hasRaisedHand(participant) && !getDisableRemoveRaisedHandOnFocus(state)) {
```

Comment:
This threshold feels a bit low. In lib-jitsi-meet we use a much higher value to detect the dominant speaker when in p2p, for example: https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/detection/P2PDominantSpeakerDetection.js#L4-L9

Reply:
changed it to the value from lib-jitsi-meet